src/accounts/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import Wallet from "./wallet";
+import axios from 'axios';
+import AddWalletModal from "../modals/addWallet";
+import Loader from "../components/shared/Loader";
+import Swal from "sweetalert2";
+
+import "./style.scss"
+
+export default function Accounts() {
+    const [walletData, setWalletData] = useState([]);
+    const [modalIsOpen, setModalIsOpen] = useState(false)
+    const [getNewData, setGetNewData] = useState(false)
+    const [isLoaded, setIsLoaded] = useState(true)
+
+    function ErrorAlert() {
+        Swal.fire({
+            icon: 'error',
+            title: 'Oops...',
+            text: 'Something went wrong!',
+            footer: '<a href="/">try Again ?</a>'
+        })
+    }
+
+    useEffect(() => {
+        axios.get(`http://localhost:3090/accounts`).then((res) => {
```

Comment:
why you added env variable in .env file if you are not using it here?

Reply:
const PORT = process.env.SERVER_PORT || 3090;    but i have a code line like this it checked if it can't find port in .env then using 3090

Reply:
I don't get what is your response to my comment, but anyway. Please create a new service and create new axios instance and add http://localhost:3090 that you will get from env as a base url